### ISSUE TITLE
[CherryPy] API client wheel_async always returns status 500

### ISSUE THREAD

**Issue by ronnievdc** [saltstack/salt#54026]
### Description of Issue

Any call to the Salt API with CherryPy using the wheel_async client will result in status code 500.

### Setup
(Please provide relevant configs and/or SLS files (Be sure to remove sensitive info).)
```
external_auth:
  pam:
    saltuser:
      - '@runner':
          - jobs.active
          - jobs.list_job
          - jobs.list_jobs
          - jobs.exit_success
          - jobs.lookup_jid
      - '@wheel':
          - key.list_all

rest_cherrypy:
  port: 3333
  host: 0.0.0.0
  disable_ssl: true
  app: /srv/salt.domain.localhost/index.html
  static: /srv/salt.domain.localhost/static
  static_path: /static
````

### Steps to Reproduce Issue
(Include debug logs if possible and relevant.)
```
curl http://0.0.0.0:3333 \
     -H 'Accept: application/x-yaml' \
     -H 'X-Auth-Token: 000000000000000000000000000000'\
     -d client=wheel_async \
     -d fun=key.list_all
```
return: An unexpected error occurred
status: 500

### Versions Report
(Provided by running `salt --versions-report`. Please also mention any differences in master/minion versions.)
```
Salt Version:
           Salt: 2019.2.0
 
Dependency Versions:
           cffi: Not Installed
       cherrypy: 3.5.0
       dateutil: 2.4.2
      docker-py: 4.0.1
          gitdb: 0.6.4
      gitpython: 1.0.1
          ioflo: Not Installed
         Jinja2: 2.8
        libgit2: Not Installed
        libnacl: Not Installed
       M2Crypto: Not Installed
           Mako: 1.0.3
   msgpack-pure: Not Installed
 msgpack-python: 0.4.6
   mysql-python: Not Installed
      pycparser: Not Installed
       pycrypto: 2.6.1
   pycryptodome: Not Installed
         pygit2: Not Installed
         Python: 2.7.12 (default, Nov 12 2018, 14:36:49)
   python-gnupg: 0.3.8
         PyYAML: 3.11
          PyZMQ: 15.2.0
           RAET: Not Installed
          smmap: 0.9.0
        timelib: Not Installed
        Tornado: 4.2.1
            ZMQ: 4.1.4
 
System Versions:
           dist: Ubuntu 16.04 xenial
         locale: UTF-8
        machine: x86_64
        release: 4.4.0-131-generic
         system: Linux
        version: Ubuntu 16.04 xenial
```

### Related issues (but does not solve the issue)
https://github.com/saltstack/salt/issues/38537
https://github.com/saltstack/salt/issues/51256

**Comment by max-arnold** [comment]
The fix (#51321) is not yet released. It will be shipped as part of Salt Neon

**Comment by dwoz** [comment]
@ronnievdc Thanks for reporting this. I am going to close it in favor of #51256
